feature/problem.feature

```diff
@@ -0,0 +1,78 @@
+Feature: Problem solving
+    A person can solve a problem by asking experts.
+
+    Scenario: a new problem creation
+        Given there is a person named "Alex"
+        When "Alex" creates a problem "How to build a decision making system"
+        And there is no similar problems yet
+        Then the problem is created
+        And the new "Route" is added to the problem
+        And the new "Round Table" is added to the route
+
+    Scenario Outline: ability to reuse a route of an already existing problem
+        Given there is a problem "How to build a decision making system"
+        And this problem has a "Route"
+        And this "Route" has a "Round Table"
+        When "Dima" creates a similar problem "How to build an automated decision making system"
+        Then he receives a suggestion to reuse the "Route" for the existing problem
+        And he receives a suggestion to reuse the "Round Table" for the existing route
+        And he makes a <decision>
+        And he gets the <result>
+        Examples:
+            | decision | result      |
+            | accept   | reuse_route |
+            | decline  | new_problem |
+
+    Scenario Outline: ability to reuse a round table of an already existing problem
```

Comment:
####  You cannot reuse a Round Table. You can reuse a Route, but not a Round Table.  
  Eg.  (1. or 2. it means Option1 or Option2)
  My problem: How to build a decision making system  
System: there is already a Route on that problem, would you like to use it?
  
1.Person: No, let me create a new Route, that one is not related to the same problem.  
2.Person: Sure, I want to reuse that old Route.  
System: That Route has 20 Round Tables, they discussed about X, Y and Z. According to your "problem", Round table 20 or Round Table 17 could be the starting point for your Round Table. Which one do you choose?  
1.Person: Sure, let's start from Round Table 17 (so we create Round Table 21th, that is attached to 17)  
2.Person: No, I would like to start from Round Table 8 (so we create Round Table 21th, that is attached to 8)

Reply:
I changed the scenario so that user can not reuse a round table anymore.